scripts/deploy_custom_upgrader.js

```diff
@@ -23,20 +23,20 @@ async function main() {
     "0xE739e93dD617D28216dB669AcFdbFC70BF95663c",
   ];
   const upgraderName = "0x4ef2f261_0xee7263da";
-
   const ModuleRegistryWrapper = await ModuleRegistry.at(config.contracts.ModuleRegistry);
   const MultiSigWrapper = await MultiSig.at(config.contracts.MultiSigWallet);
   const multisigExecutor = new MultisigExecutor(MultiSigWrapper, deploymentAccount, config.multisig.autosign);
 
   const UpgraderWrapper = await Upgrader.new(
     modulesToRemove,
     modulesToAdd,
+    { gas: 1200000 }
```

Comment:
Why are custom gas limits necessary here?

Reply:
Because the default of 6000000 would take forever to get mined on mainnet.